uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/AuthenticationView.qml

```diff
@@ -26,6 +26,7 @@ import QtQuick.Controls
   \since Esri.ArcGISRuntime 100.10
   \brief A view for handling authentication challenges and automatically
   launching the appropriate UI for each type of authentication.
+  \deprecated [200.8] Use Authenticator instead.
```

Comment:
I assume we want to deprecate this? I know the Authenticator technically invokes it on the back end if using legacy auth, but still, users should be moving away in prep for 300

Reply:
Yes, I think we should deprecated it.

Reply:
Also, this raises the question if we should deprecate all the other "legacy" views. I think we should.

Reply:
i just double checked those - we don't have any qdoc for those. I think we intended for people to directly use the AuthenticationView and not the child views it contained. So probably nothing to do there.